arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -571,6 +575,53 @@ def test_plot_pair_overlaid(models, kwargs):
     assert ax is ax2
     assert ax.shape
 
+@pytest.mark.parametrize(
+    "kwargs", [{"var_names": ["mu", "theta"]}]
+)
+def test_plot_pair_combinedims(models, kwargs):
+    ax = plot_pair(models.model_1, combine_dims=["school"], **kwargs)
+    assert np.all(ax)
```

Comment:
This test can be added as an extra case in `test_plot_pair`, it will test the same it is doing here. It may be even better to add the `combine_dims=["school"]` to one of the many already present cases.